### DISCUSSION TITLE
Docs: request GitHub API infinitely for external project links

### DISCUSSION THREAD

**Discussion by yanyongyu** [fastapi/fastapi#10523] [Questions] [answered]
### First Check

- [X] I added a very descriptive title here.
- [X] I used the GitHub search to find a similar question and didn't find it.
- [X] I searched the FastAPI documentation, with the integrated search.
- [X] I already searched in Google "How to X in FastAPI" and didn't find any information.
- [X] I already read and followed all the tutorial in the docs and didn't find an answer.
- [X] I already checked if it is not related to FastAPI but to [Pydantic](https://github.com/pydantic/pydantic).
- [X] I already checked if it is not related to FastAPI but to [Swagger UI](https://github.com/swagger-api/swagger-ui).
- [X] I already checked if it is not related to FastAPI but to [ReDoc](https://github.com/Redocly/redoc).

### Commit to Help

- [X] I commit to help with one of those options 👆

### Description

- Open the docs link: https://fastapi.tiangolo.com/external-links/#projects
- see the github api requests in the dev tool
- when github api rate limit exceeded, still requests the next page

The problem is here, response status is not checked: https://github.com/tiangolo/fastapi/blob/2754d4e0fe2e45673d4ded8491a40f6d1bcd1d80/docs/en/docs/js/custom.js#L3-L21

### Operating System

Linux

### FastAPI Version

Any

### Pydantic Version

Any

### Python Version

Any

**Comment by alejsdev** [comment, marked as answer]
Thanks for your feedback @yanyongyu 
This was already handled and now it's working :nerd_face:
